pkg/subctl/cmd/show_versions.go

```diff
@@ -120,16 +125,10 @@ func showVersions(cmd *cobra.Command, args []string) {
 	for _, item := range configs {
 		fmt.Println()
 		fmt.Printf("Showing information for cluster %q:\n", item.clusterName)
-		versions := getVersions(item.config)
-		printVersions(versions)
+		getVersions(item.config)
```

Comment:
bubbling up the errors means that err gets at least up to here, and here you print or exit, depending on how hard the error is.